BackendApartmentReservation/BackendApartmentReservation/Controllers/ProfilesController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BackendApartmentReservation.Controllers
+{
+    [Route("api")]
+    [ApiController]
```

Comment:
Is `[ApiController]` the same as inheriting from `ControllerBase`?

Reply:
Pretty sure `ControllerBase` should be inherited from

Reply:
Will add, thanks 😄